frontend/compositions/map/BaseMap.tsx

```diff
@@ -0,0 +1,46 @@
+import * as d3 from "d3"
+import { FeatureCollection } from "geojson"
+import React, { useEffect, useRef } from "react"
+import * as topojson from "topojson"
+import { Topology } from "topojson-specification"
+import styles from "./map.module.css"
+
+type BaseMapProps = {
```

Comment:
We shouldn't be mixing type aliases into the codebase since we've decided on interfaces: This and any other type aliases in this ticket should be converted to interfaces.